reader_test.go

```diff
@@ -248,44 +229,44 @@ type TestPointerType struct {
 	Utf8String *string  `maxminddb:"utf8_string"`
 }
 
-func (s *MySuite) TestComplexStructWithNestingAndPointer(c *C) {
+func TestComplexStructWithNestingAndPointer(t *testing.T) {
 	reader, err := Open("test-data/test-data/MaxMind-DB-test-decoder.mmdb")
-	c.Assert(err, IsNil)
+	assert.Nil(t, err)
 
 	var result TestPointerType
 
 	err = reader.Lookup(net.ParseIP("::1.1.1.0"), &result)
-	c.Assert(err, IsNil)
+	assert.Nil(t, err)
 
-	c.Assert(*result.Array, DeepEquals, []uint{uint(1), uint(2), uint(3)})
-	c.Assert(*result.Boolean, Equals, true)
-	c.Assert(*result.Bytes, DeepEquals, []byte{0x00, 0x00, 0x00, 0x2a})
-	c.Assert(*result.Double, Equals, 42.123456)
-	c.Assert(*result.Float, Equals, float32(1.1))
-	c.Assert(*result.Int32, Equals, int32(-268435456))
+	assert.True(t, assert.ObjectsAreEqual(*result.Array, []uint{uint(1), uint(2), uint(3)}))
+	assert.Equal(t, *result.Boolean, true)
+	assert.True(t, assert.ObjectsAreEqual(*result.Bytes, []byte{0x00, 0x00, 0x00, 0x2a}))
+	assert.Equal(t, *result.Double, 42.123456)
+	assert.Equal(t, *result.Float, float32(1.1))
+	assert.Equal(t, *result.Int32, int32(-268435456))
 
-	c.Assert(result.Map.MapX.ArrayX, DeepEquals, []int{7, 8, 9})
+	assert.True(t, assert.ObjectsAreEqual(result.Map.MapX.ArrayX, []int{7, 8, 9}))
```

Comment:
Couldn't this be just written as `assert.Equal(t, result.Map.MapX.ArrayX, []int{7, 8, 9})`? `assert.Equal` [appears](https://github.com/stretchr/testify/blob/master/assert/assertions.go#L256) to just call `assert.ObjectsAreEqual` internally. This also applies to the other `ObjectsAreEqual` calls.